runtime/runtime.go

```diff
@@ -1732,6 +1732,19 @@ func (r *interpreterRuntime) newAuthAccountContractsChangeFunction(
 				))
 			}
 
+			if isUpdate {
+				validator := NewContractUpdateValidator(existingCode, program.Program)
+				err := validator.Validate()
+				if err != nil {
+					panic(&InvalidContractDeploymentError{
+						Err: fmt.Errorf("cannot update contract `%s` in account %s: %s",
+							nameArgument,
+							address.ShortHexWithPrefix(),
+							err.Error()),
```

Comment:
`fmt.Errorf` has a special formatting directive for nested errors, `%w`:

```suggestion
						Err: fmt.Errorf("cannot update contract `%s` in account %s: %w",
							nameArgument,
							address.ShortHexWithPrefix(),
							err,
						),
```